snipsmanagercore/server.py

```diff
@@ -49,17 +63,12 @@ def __init__(self,
         self.client.on_message = self.on_message
         self.mqtt_hostname = mqtt_hostname
         self.mqtt_port = mqtt_port
+        self.tts_service_id = tts_service_id
+        self.locale = locale
+        self.dialogue = SnipsDialogueAPI(self.client, tts_service_id, locale)
 
-        if tts_service_id == "google":
-            self.tts_service = GTTS(locale, logger=self.logger)
-        else:
-            self.tts_service = SnipsTTS(
-                self.thread_handler,
-                mqtt_hostname,
-                mqtt_port,
-                "hermes/tts/say",
-                locale,
-                logger=self.logger)
+        if tts_service_id not in ["snips", "google", None]:
+            self.log_error("Warning ! We only support Snips or Google TTS.")
```

Comment:
Do we need to show an error here? Fine to create assistants without TTS.

Also, no space before "!" in English :)

Reply:
No need for this warning anymore! :)